client/app/components/packages/applicant-fieldset.hbs

```diff
@@ -36,12 +36,10 @@
 
     {{!-- and if the applicant is on behalf of an organization, we need to know which organization --}}
     {{#if (eq @applicant.dcpType 717170001)}}
-      <Ui::Question data-test-applicant-organization as |Q|>
-        <Q.Prompt>
-          <span class="text-weight-normal">Organization Name</span>
-        </Q.Prompt>
-        <Form.Field @attribute="dcpOrganization"/>
-      </Ui::Question>
+      <label data-test-applicant-organization>
+        Organization Name <Ui::RequiredAsterisk />
+        <Form.Field @attribute="dcpOrganization" />
+      </label>
```

Comment:
Why are we using just <label> here, and not Ui::Question and Q.label?

Reply:
The main benefit of `Ui::Question` is to standardize our markup, avoid duplicating classes that style labels, and to avoid labels wrapping inputs (by leveraging `for=""`) so that questions can contain complex/interactive UI (help text, additional guidance, validation, counter). 

In order to give visual hierarchy to the applicant info, its inputs are not styled the same as top-level questions. And they don't require any classes, or contain any complex help text or additional guidance. They're much simpler. 

In order to use `Ui::Question` for these simple inputs, we'd need to complicate the component with a way to conditionally style questions — with regular instead of bold labels, and without margin between the questions. 

I'd rather have these few inputs done differently than overcomplicate the `Ui::Question` API.

Reply:
Okay 👍 Thanks for the explanation! I really think we should have documentation laying out the different uses of these components

Reply:
I'll create an issue for that